client/scss/utilities/_root-scope-classes.scss

```diff
@@ -18,6 +30,19 @@
   margin-bottom: $spacing-unit * 2;
 }
 
+.l-margin-s {
+  margin-left: $spacing-unit * 2;
+}
+
+.r-margin-s {
+  margin-right: $spacing-unit * 2;
+}
+
+.h-center {
```

Comment:
It is a bit of a stylistic thing - but personally once I started using CSS building things became way easier.